tests/span/Port-mirroring-test-plan.md

```diff
@@ -0,0 +1,119 @@
+# Port mirroring test plan
+
+## Rev 0.1
+
+- [Revision](#revision)
+- [Definition/Abbrevation](#definition/abbrevation)
+- [Overview](#overview)
+  - [Scope](#scope)
+  - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+  - [test_port_mirroring_rx](#Test-case-test_port_mirroring_rx)
+  - [test_port_mirroring_tx](#Test-case-test_port_mirroring_tx)
+  - [test_port_mirroring_rx_tx](#Test-case-test_port_mirroring_rx_tx)
+  - [test_port_mirroring_from_multiple_ports](#Test-case-test_port_mirroring_from_multiple_ports)
+
+## Revision
+
+| Rev |     Date    |       Author            |     Change Description      |
+|:---:|:-----------:|:------------------------|:----------------------------|
+| 0.1 |  05/02/2021 | Intel : Viktor Cheketa  |       Initial version       |
+
+## Definition/Abbrevation
+
+| **Term**     | **Meaning**                            |
+|--------------|----------------------------------------|
+|    SPAN      | Switched Port ANalyzer                 |
+| Monitor port | Destination port of mirroring session  |
+| Mirrored port| Source port of mirroring session       |
+
+## Overview
+
+The purpose is to test the functionality of port mirroring (SPAN) feature on SONiC switch DUT.
+
+## Scope
+
+The test is targeting a runninc SONiC system with fully functioning configuration.
+Purpose of test is to verify that a SONiC switch system correctly performs port mirroring implementation
+based on configured sessions.
+
+## Testbed
+
+Supported topologies: t0
+
+## Setup configuration
+
+No setup pre-configuration is required, test will configure and clean-up all the configuration.
+
+### Setup of DUT switch
+
+Each test creates a mirroring session with specific parameters. In order to create a session, monitor port must not
+be a VLAN member.
```

Comment:
Should the tests be run with both a LAG and physical monitor port then?

Reply:
Could you elaborate a little?
Did you mean sending traffic from LAG to physical port instead of from one physical port to another?
Or did you suggest extending these tests with LAG support (i.e. using LAG as source of mirroring session)?

Reply:
LAG as source of mirroring session.